packages/rxjs-firebase/src/operators/documentSnapshotStateObservable.ts

```
@@ -0,0 +1,14 @@
+import { type DocumentData } from '@firebase/firestore'
```

ESLint

[error] Filename is not in kebab case. Rename it to `document-snapshot-state-observable.ts`.

(unicorn/filename-case)